peepingtom/io_/read/mrc/mrc.py

```diff
@@ -10,6 +10,6 @@ def read_mrc(image_path, name_regex=None, **kwargs):
     read an mrc file and return an ImageBlock
     """
     name = guess_name(image_path, name_regex)
-    mrc = mrcfile.open(image_path)
+    mrc = mrcfile.mmap(image_path)
```

Comment:
I would make this into an if-else, and add an option to `read_mrc` on the lines of `mmap=False` to control the flow. This way, it can be passed from `pt.peep` and it should already trickle down as appropriate.